net-analyzer/greenbone-security-assistant/greenbone-security-assistant-20.8.1.ebuild

```diff
@@ -0,0 +1,121 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_MAKEFILE_GENERATOR="emake"
+inherit cmake flag-o-matic systemd toolchain-funcs
+
+MY_PN="gsa"
+MY_DN="gsad"
+MY_NODE_N="node_modules"
+
+DESCRIPTION="Greenbone Security Assistant"
+HOMEPAGE="https://www.greenbone.net/en/"
+SRC_URI="https://github.com/greenbone/${MY_PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+https://github.com/greenbone/${MY_PN}/releases/download/v${PV}/gsa-node-modules-${PV}.tar.gz -> ${P}-${MY_NODE_N}.tar.gz"
+
+SLOT="0"
+LICENSE="GPL-2+"
+KEYWORDS="~amd64 ~x86"
+IUSE="extras"
+
+DEPEND="
+	dev-libs/libgcrypt:0=
+	dev-libs/libxml2
+	dev-libs/libxslt
+	>=net-analyzer/gvm-libs-20.8.0
+	net-libs/gnutls:=
+	net-libs/libmicrohttpd"
+
```

Comment:
As I see `fowners gvm:gvm` below, please state in the ebuild that they are needed by depending on correct *acct-\*/gvm* ebuilds.